src/_includes/partials/global/footer.njk

```diff
@@ -9,6 +9,7 @@
         <p class="h3">Social media</p>
         <ul>
             <li><a rel="external" href="https://twitter.com/{{ site.twitter | replace('@', '') }}/">{% include 'svg/twitter.svg' %} <span class="underline">Twitter</span></a></li>
+            <li><a rel="external" href="https://bsky.app/profile/{{ site.bluesky | replace('@', '') }}/">{% include 'svg/bluesky.svg' %} <span class="underline">Bluesky</span></a></li>
```

Comment:
I wonder, if there's any value in setting the site data value as `@idrc-ocadu.bsky.social`; if we just set site data value to include the url part, we don't have to use this replace filter. I see this pattern has used for few different social medias, like twitter and medium, but I think we can just simplify these blocks with full urls stored in the data file. If you think this separation is useful, this PR looks good.

Reply:
I understand what you're saying because we don't seem to use those outside of the URLs. I wonder if the intention was to allow for them to be used in other ways as well. I think maybe we can file that as a separate issue to investigate and assign it to @greatislander for when he's back. Then we can keep track of it and also determine if there is another use case.

Reply:
Sounds good :)